apps/api_web/lib/api_web/controllers/prediction_controller.ex

```diff
@@ -261,6 +261,34 @@ defmodule ApiWeb.PredictionController do
               example: "2017-08-14T15:38:58-04:00"
             )
 
+            arrival_uncertainty(
```

Comment:
Does this get sucked into Swagger? I can't remember, but if it doesn't, might be worth outputting this to the docs wherever this field shows up.